app/controllers/api/v1/admin/base_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::Admin::BaseController < ApplicationController
+  before_action :require_admin
```

Comment:
this controller allows me to reuse this method but keeps it out of the application controller